Check title failed but the unknown works already in cspell.config.yml

```bash
npm warn exec The following package was not found and will be installed: cspell-cli@9.7.0
1/1  730.21ms X
:1:20 - Unknown word (setuptools)
:1:47 - Unknown word (pyproject)
CSpell: Files checked: 1, Issues found: 2 in 1 file.
Error: Process completed with exit code 1.
```

It's a weird to see a failed. but actually, the unknown words in the [cspell.config.yml](https://github.com/cpp-linter/.github/blob/main/cspell.config.yml).

https://github.com/cpp-linter/cpp-linter-hooks/actions/runs/23263124309/job/67637095130

Any throughs? @2bndy5

_Originally posted by @shenxianpeng in https://github.com/cpp-linter/cpp-linter-hooks/issues/188#issuecomment-4085105722_

--- a/.github/workflows/pre-commit.yml
+++ b/.github/workflows/pre-commit.yml
@@ -85,5 +85,10 @@ jobs:
       - name: spell check
         env:
           PR_TITLE: "${{ steps.get-title.outputs.title }}"
-          CSPELL_CONFIG: ${{ github.workspace }}/org-repo/cspell.config.yml
-        run: echo "${PR_TITLE}" | npx cspell-cli lint --config "${CSPELL_CONFIG}" stdin
+        run: |
+          if [ -f project-repo/cspell.config.yml ]; then
+            CSPELL_CONFIG="project-repo/cspell.config.yml"
+          else
+            CSPELL_CONFIG="org-repo/cspell.config.yml"
+          fi
+          echo "${PR_TITLE}" | npx cspell-cli lint --config "${CSPELL_CONFIG}" stdin

--- a/.github/workflows/pre-commit.yml
+++ b/.github/workflows/pre-commit.yml
@@ -83,7 +83,7 @@ jobs:
         with:
           node-version: latest
       - name: spell check
-        working-directory: project-repo
         env:
           PR_TITLE: "${{ steps.get-title.outputs.title }}"
-        run: echo "${PR_TITLE}" | npx cspell-cli lint stdin
+          CSPELL_CONFIG: ${{ github.workspace }}/org-repo/cspell.config.yml
+        run: echo "${PR_TITLE}" | npx cspell-cli lint --config "${CSPELL_CONFIG}" stdin

